src/js/controllers/mapController.ts

```diff
@@ -901,6 +901,20 @@ export class MapController {
     store.dispatch(renderModal(''));
   }
 
+  updateDensityValue(value: number): void {
+    densityEnabledLayers.forEach((layerId: string) => {
+      const layer: any = this._map?.findLayerById(layerId);
+      if (layer) {
+        console.log(layer);
```

Comment:
Should we remove the `console.log()` or is it a good fit to leave it in for debugging? Double-checking since there are some projects to log layer info for debugging purposes